components/speedreader/resources/speedreader-desktop.css

```diff
@@ -82,6 +87,7 @@ a[href] {
 #article {
   text-rendering: optimizeLegibility;
   font-family: "Georgia", "Times New Roman", Times, serif;
+  user-select: unset;
```

Comment:
why are we disabling user's ability to select text in the article?

Reply:
I took the entire speedreader-desktop.css file from the downloadable components. I don't know why :/

Reply:
UPD: This rule doesn't disable text selection. It should be 'none' to disable, am i wrong ?

Reply:
oh, right. still not sure which value it's unsetting from or why it's needed